web/themes/custom/pineapple/sass/components/_teaser-collection.scss

```diff
@@ -13,13 +13,14 @@
     width: 100%;
 
     &.vertical {
-      margin-bottom: $unit * 3;
-      padding-right: 15px;
+      margin-bottom: $unit * 2;
       padding-left: 15px;
+      padding-right: 15px;
 
       @include breakpoint($lap) {
         flex: 0 0 50%;
         max-width: 50%;
+        padding-right: 0;
```

Comment:
This is not the correct way to fix the miss alignment bug, you need to add a negative margin to the right as you've done on the left for the teaser collection container.

Reply:
done